package.json

```diff
@@ -68,6 +68,7 @@
     "eslint-plugin-node": "^5.1.0",
     "eslint-plugin-prettier": "^2.1.2",
     "jest": "^20.0.4",
+    "jest-cli": "^20.0.4",
```

Comment:
Actually I added it because I ran the tests with `npm test` and node complained that `jest-cli` is not in the project